spaceapi_server/Cargo.toml

```diff
@@ -10,6 +10,7 @@ authors = [
 redis = "0.3"
 rustc-serialize = "0.3"
 iron = "0.1"
+urlencoded = "*"
```

Comment:
Oh, nice, I didn't know that ^ stands for SemVer :)

I agree about the `^0.2`, altough the SemVer semantics are active by default. So strictly speaking the `^` isn't necessary. But it makes the requirement more obvious.